Automattic/wp-calypso
# Optimize images

In this PR I made some optimizations and cosmetic updates to the illustrations so that the progress indicator is the main focal point of the banner. 

## Before
![screen shot 2017-12-07 at 10 24 26 pm](https://user-images.githubusercontent.com/6981253/33773204-1b0ee67a-dc05-11e7-8bf0-8197647905de.png)

## After
![image](https://user-images.githubusercontent.com/6981253/33773339-8f8fd036-dc05-11e7-957c-c062ec0e7ba0.png)

## Testing
* Create a new site
* View the banners on stats page
* Visit checklist to mark tasks completed
* Return to the stats page to see another banner

cc @taggon
